bugsnag-android-core/src/main/java/com/bugsnag/android/Metadata.kt

```diff
@@ -145,9 +144,8 @@ internal data class Metadata @JvmOverloads constructor(
         }
     }
 
-    fun copy() = this.copy(
-        store = toMap(),
-        jsonStreamer = jsonStreamer,
-        redactedKeys = redactedKeys
-    )
+    fun copy(): Metadata {
+        return this.copy(store = toMap())
+            .also { it.redactedKeys = redactedKeys }
```

Comment:
Doesn't this want to be a copy of the redactedKeys set?

Reply:
In practice a copy isn't required because nothing creates an iterator on the object anymore. For consistency with `objectJsonStreamer` however I've updated this to make a copy of the set.